README.md

```diff
@@ -7,3 +7,14 @@ devices in the cloud.
 The application is developed for and tested on Google Cloud Platform, but it's
 designed in a way that should allow its deployment in other Cloud solutions
 without much trouble.
+
+## Links for each component
+
+To know details, please refer to these documents below.
+
+- [Cloud Orchestrator](docs/cloud_orchestrator.md): A web service for hosting
+VMs or containers for running Cuttlefish instances on top of.
+- [cvdr](docs/cvdr.md): CLI wrapper of Cloud Orchestrator providing
```

Comment:
`cvdr` is not a wrapper. it could be rephrased as a `CLI client`

Reply:
https://github.com/google/cloud-android-orchestration/pull/321